src/components/ProductView/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import ProductContent from './product-content'
+import ProductDetail from './product-detail'
+import ProductInformation from './product-information'
+
+const ProductView = () => {
+  return (
+    <section className="product-view">
+      <div className="product-view__content">
+        <ProductContent />
+      </div>
+      <div className="product-view__detail">
+        <ProductDetail />
+      </div>
+      <div className="product-view__information">
+        <ProductInformation />
+      </div>
+    </section>
```

Comment:
What if we move these block `(product-view_...)` into its own component instead. Do you think it looks cleaner?
```suggestion
    <section className="product-view">
      <ProductContent />
      <ProductDetail />
      <ProductInformation />
    </section>
```

Reply:
This must still be contained in a `div` since they are separated by `grid` display in `product-view`. Do you agree? @kimvcp